js/modules/k6/timers/timers.go

```diff
@@ -62,10 +62,12 @@ func (*RootModule) NewModuleInstance(vu modules.VU) modules.Instance {
 func (e *Timers) Exports() modules.Exports {
 	return modules.Exports{
 		Named: map[string]interface{}{
-			"setTimeout":    e.setTimeout,
-			"clearTimeout":  e.clearTimeout,
-			"setInterval":   e.setInterval,
-			"clearInterval": e.clearInterval,
+			// TODO the usage of `ToValue` here is so that goja doesn't do it automatically later
```

Comment:
Nit; I don't fully get what's pending "to do" here based on this comment 🤔

Reply:
This is a workaround for https://github.com/grafana/k6/issues/3707 

Maybe TODO isn't the best option here :thinking: